src/chapters/sql-part2/creating-databases.rst

```diff
@@ -0,0 +1,149 @@
+   `practice data <https://gist.github.com/jimflores5/795a7faae8d35d36033872ae30b88328>`__.
+#. As you did in the first :ref:`SQL studio <movie-sqls>`, download the zip
+   file. Double-click on it to extract three ``.csv`` files.
+#. In MySQL Workbench, right-click on the ``writing_supply`` table. Select
+   *Table Data Import Wizard*.
+
+   .. figure:: ./figures/SQLWorkbenchImport.png
+      :alt: Table Data Import Wizard menu option.
+
+#. In the next panel, click the *Browse* button. Find and select the
+   ``writing_supply.csv`` file. Click *Next*.
+
+   .. figure:: ./figures/importCsvFile.png
+      :alt: Select file path to csv file.
+
+#. In the following panel, select the default option
+   *Use existing table --> storage.writing_supply*, and click *Next*.
+#. Once again, accept the default options for importing data into your table
+   columns. Click *Next*.
+
+   .. figure:: ./figures/importColumns.png
+      :alt: Select columns to import.
+
+#. Clicking *Next* again will import the data. Clicking *Finish* on the final
+   panel returns you to the editor.
+#. In the query tab, run ``SELECT * FROM writing_supply`` to confirm that 6
+   entries now exist in the table.
+#. Repeat the steps 3 - 8 for the ``pencil_drawer`` and ``pen_drawer`` tables.
+
+Confirm that the ``pencil_drawer`` and ``pen_drawer`` tables hold 6 and 15
+entries, respectively. You are now ready to practice more advanced SQL queries.
+
+Import Troubleshooting
+----------------------
+
+Did something go wrong with your import? No worries---the process is not always
+perfect, and errors occur more often than we would like. Some issues are just
+random glitches, while others involve syntax or selection mistakes.
+
+Here are some common errors:
+
+#. **Filename**: Make sure that each file ends with the ``.csv`` extension.
+   Also, select the correct file to import into each table (e.g.
+   ``pen_drawer.csv`` will not import into the ``writing_supply`` table).
+#. **Mismatched data types**: MySQL Workbench will throw an error if you define
+   a table column as one data type, but the matching column in the ``.csv``
+   file contains a different data type. Check to make sure that your ``CREATE``
+   statements correctly define the data types described in the
+   :ref:`Setup <practice-setup>` section.
+#. **Import order**: For one-to-many relationships, the *one* table must be
+   created before the *many*, and this idea also applies to filling the tables
+   with data. ``pen_drawer`` and ``pencil_drawer`` both contain foreign keys
+   that link to ``writing_supply``. Thus, you must import data into
+   ``writing_supply`` FIRST, before attempting to add data to the other two
+   tables.
+
+.. admonition:: Note
+
+   This is NOT a complete list of everything that could happen to prevent a
+   successful import. Sorry.
+
+   If your issue is not discussed above, YAY! You get to practice your
+   real-world developer skills by figuring out how to make it work!
```

Comment:
This note should be reworded to make it more comforting. Maybe simply, "These are ways to address common errors and this is not an exhaustive list of everything that can go wrong with a data import. Don't hesitate to consult the MySQL documentation or your instructor!"

Reply:
Changes applied.  Thanks.